resnet/yamls/mcloud_run.yaml

```diff
@@ -0,0 +1,95 @@
+run_name: resnet_example
+cluster: null  # Name of the cluster to use for this run
+gpu_type: null # Type of GPU to use
+gpu_num: null  # Number of GPUs to use specified as an integer
+image: mosaicml/pytorch:1.12.1_cu116-python3.9-ubuntu20.04
+integrations:
+  - integration_type: "git_repo"
+    git_repo: mosaicml/benchmarks
+    git_branch: landan/resnet_benchmark
+command: |
+  cd benchmarks/resnet
+  pip install -r requirements.txt
+  composer main.py /mnt/config/parameters.yaml
+
+# Configuration copied from baseline.yaml
+parameters:
```

Comment:
Can this just not be here? Single specification on config would be a big readability, usability, and maintainability win.

Reply:
How are you thinking to reference `baseline.yaml` in this yaml? The other option in my mind is to do `composer main.py yamls/resnet50.yaml`. But this would mean that you would need to commit and push any changes to `resnet50.yaml` which I would find very annoying if I wanted to run something quickly.

I agree it would make readability and maintainability easier, but it seems like a big hit to usability.